routes/cartRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const cartController = require('../controllers/cartController');
+const loginReq = require('../utils/auth');
```

Comment:
```suggestion
const express = require('express');

const cartController = require('../controllers/cartController');
const loginReq = require('../utils/auth');
```
각 모듈간 구분을 위해 외부 묘듈, 커스텀 모듈 사이를 줄바꿈으로 구분지어주면 더 좋을 것 같습니다~!